tools/CouchDB_Import_MRI.php

```diff
@@ -65,59 +65,100 @@ function updateDataDict($types)
 
         foreach ($types as $type) {
             $ScanType = $type['ScanType'];
-            $this->Dictionary["Selected_$ScanType"] = array(
-                'Type' => 'varchar(255)',
-                'Description' => "Selected $ScanType file for session",
-                'IsFile' => true
-            );
-            $this->Dictionary[$ScanType . "_QCStatus"] = array(
-                'Type' => "enum('Pass', 'Fail')",
-                'Description' => "QC Status for $ScanType file"
-            );
+
+            $SelectedArray = array(
+                              'Type'        => 'varchar(255)',
+                              'Description' => "Selected $ScanType file for session",
+                              'IsFile'      => true,
+                             );
+            $QCStatusArray = array(
+                              'Type'        => "enum('Pass', 'Fail')",
+                              'Description' => "QC Status for $ScanType file",
+                             );
+
+            $this->Dictionary["Selected_$ScanType"]    = $SelectedArray;
+            $this->Dictionary[$ScanType . "_QCStatus"] = $QCStatusArray;
             $feedback_types = $mri_feedback->getAllCommentTypes();
             foreach ($feedback_types as $CommentTypeID=>$comments) {
                 if (!empty($comments['field'])) {
                     $fieldName = $comments['field'];
                     $type      = "enum ('" .implode("','", $comments['values'])."')";
 
-                    $this->Dictionary["Comment_".$fieldName."_$ScanType"] = array(
-                        'Type' => 'varchar(255)',
-                        'Description' => "Overall Comment for $fieldName $ScanType"
-                        );
+                    $cmt_field = "Comment_".$fieldName."_$ScanType";
```

Comment:
We should probably make this script consistent with how it names things. Starting with $ScanType_$FieldName would make sorting easier. (All the scantype fields would show up together in the DQT.)